### ISSUE TITLE
Babel 6.4.3 breaks with syntax error "read-only"

### ISSUE THREAD

**Issue by hsiliev** [cloudfoundry-incubator/cf-abacus#200]
With today's (Jan 14th 2016) release of Babel 6.4.3 there are a lot of errors about read-only const variables.

For example one of the errors reads:
```
> abacus-usage-accumulator@0.0.4 babel /Users/development/workspace/cf-abacus/lib/aggregation/accumulator
> babel

SyntaxError: src/index.js: "cc" is read-only (This is an error on an internal node. Probably an internal error. Location has been estimated.)
```

The same problem is reported here: https://github.com/babel/babel.github.io/issues/679 and https://phabricator.babeljs.io/T2892

The second link pinpoints the problem to the preset2015 dependency.

**Comment by hsiliev** [comment]
The following workaround enabled the build of abacus:
``
npm run clean
npm install "babel-types@<=6.4.2"
npm install "babel-plugin-transform-es2015-typeof-symbol@<=6.4.2"
npm install "babel-plugin-transform-es2015-unicode-regex@<=6.4.2"
npm install "babel-plugin-transform-regenerator@<=6.4.2"
npm install "babel-generator@<=6.4.2"
npm install "babel-register@<=6.4.2"
npm run build
```

**Comment by jsdelfino** [comment]
@hsiliev Thanks for finding this out! Looks like the Babel folks are going to fix this (see https://phabricator.babeljs.io/T2892). In the meantime your work around helps!

**Comment by hsiliev** [comment]
The problem seems fixed in 6.4.4
